docs/modules/migrate/pages/data-migration-tool.adoc

```diff
@@ -0,0 +1,176 @@
+. Verify the data in the target cluster.
+
+=== Step 1. Start the Source Cluster
+
+Run the following Docker command to start a one-member cluster.
+
+[source,shell]
+----
+$ docker run -p 127.0.0.1:5701:5701 -e HZ_CLUSTERNAME=source hazelcast/hazelcast:4.x.y
+----
+
+NOTE: For the `4.x.y` placeholder, you can use one of the released versions from Hazelcast 4 series, e.g., `4.2.7`.
+
+This member is started using an internal Docker IP and the port `5701` on that IP address, something like `172.12.0.1:5701`.
+Since the Docker container is in a separate network, it does not collide with the running processes on your
+local machine. For example, in this case, the member can run on `127.0.0.1:5701`, and our container can run on `172.12.0.1:5701`.
+
+However, since it runs in a virtual network, we need to make it accessible to your local processes because migration and target
+clusters, CLC, and DMT will be processes outside the Docker environment, directly running on your computer. The `-p` option in the Docker command allows you to 
+map a container's port to the host machine. The command assigns `5701` as the container's port, so `127.0.0.1:5701` becomes the Hazelcast member's address.
+`127.0.0.1` refers to your local machine's address. Make sure there is nothing else running on port `5701` on your local machine.
+
+NOTE: Instead of using Docker to start a member, you can also download one of the 4.x.y packages from https://hazelcast.com/open-source-projects/downloads/archives/#hazelcast-imdg[Hazelcast web site] and https://docs.hazelcast.com/imdg/4.2/getting-started[start a member]. Make sure to https://docs.hazelcast.com/imdg/4.2/clusters/creating-clusters[set the cluster's name] as `source`.
+
+=== Step 2. Create a Map and Put Data
+
+The `source.yaml`, which is contained in the DMT download package, is the Hazelcast CLC configuration file to access the source cluster to populate data.
+You can modify the `address` field in this file, if needed, in case you used a different port mapping than `5701` while starting the member, 
+or you started a cluster without using Docker and it started on a port other than `5701`.
+
+The content of `source.yaml`:
+
+[source,yaml]
+----
+cluster:
+  name: "source"
+  address: "127.0.0.1:5701"
+----
+
+Check that you can put an entry to the source cluster by running the following CLC command on your terminal.
+
+[source,shell]
+----
+clc -c source.yaml map --name my-map set key-1 value-1
+----
+
+??? _Any possible errors and ways of fixing them here_ ???
+
+Then, you can run the following script on your terminal to populate 1000 entries to the source cluster. This script is 
+for macOS and Linux.
+
+[source,shell]
+----
+for i in {1..1000}; do clc -c source.yaml map --name my-map set key-$i value-$i --quiet; done && echo OK
+----
+
+This one is for Windows users:
+
+[source,shell]
+----
+for /l %x in (1, 1, 100) do clc -c source.yaml map --name my-map set key-%x value-%x --quiet
```

Comment:
to be honest, I haven't tried this one actually. Let me do that as well. I have a mac and I don't want to install windows on it. Probably will do it on my desktop. Will do it tomorrow probably

Reply:
lets make this 1000 entries instead of 100